tqecjs/src/workspace/plaquettes/PlaquetteClass.js

```diff
@@ -0,0 +1,370 @@
+import { Graphics, Container } from 'pixi.js';
+import { button } from '../components/button';
+
+export default class Plaquette extends Graphics {
+	constructor(qubits, workspace, gridSize = 50, color = 'purple') {
+		super();
+		// UI properties
+		this.workspace = workspace;
+		this.color = color;
+		this.gridSize = gridSize;
+		this.gridOffsetX = this.gridSize;
+		this.gridOffsetY = this.gridSize;
+		this.isDragging = false;
+		this.plaquetteMade = false;
+		this.showButton = false;
+		this.controlPanel = new Container();
+		this.controlPanel.name = 'control_panel';
+		this.controlPanel.visible = false;
+		this.rotateButton = button('Rotate', 200, 200, 0x000000);
```

Comment:
or `'black'` but I am bad at remembering hex colors ;-)

Reply:
Updated all hexes to use literal name